locales/fr.json

```diff
@@ -12,15 +12,15 @@
     "A collection of posts": "Une catégorie d'articles",
     "A collection of 1 post": "Une catégorie avec un article",
     "A collection of % posts": "Une catégorie avec % articles",
-    "Get the latest posts delivered right to your inbox": "Recevez les derniers articles directement dans votre boîte de réception.",
+    "Get the latest posts delivered right to your inbox": "Recevez les derniers articles directement dans votre boîte aux lettres.",
```

Comment:
Thank you. Not sure -I'm not French-, but it seems [boîte de réception](https://www.google.es/search?q=bo%C3%AEte+de+r%C3%A9ception&tbm=isch) is very used for inbox folder (Gmail, etc..) while [boîte aux lettres](https://www.google.es/search?q=boîte+aux+lettres&tbm=isch) refers more often to physical mail. The other corrections seem OK.

Reply:
All right, I've seen [boîte aux lettres électronique](https://fr.wikipedia.org/wiki/Bo%C3%AEte_aux_lettres_%C3%A9lectronique) or BALE is also used. I merge it. Thank you!